asterisk/agi/phpstan-baseline.neon

```diff
@@ -55,6 +55,11 @@ parameters:
 			count: 1
 			path: src/Agi/Action/ExternalFilterAction.php
 
+		-
+			message: "#^Cannot call method getName\\(\\) on Ivoz\\\\Provider\\\\Domain\\\\Model\\\\Company\\\\CompanyInterface\\|null\\.$#"
```

Comment:
We should add some comment like to let phpstan know that this var is not nullable in this context
```
/** @var CompanyInterface $company */
$company = $this->getCompany()
```